roles/ovirt-engine-setup/templates/answerfile_4.1_upgrade.txt.j2

```diff
@@ -0,0 +1,72 @@
+# action=setup
```

Comment:
Why do we need a special templates for new installation and upgrades?

Reply:
Because for upgrading you have some extra questions when running the engine setup.

Reply:
OK, makes sense